programs/cleos/main.cpp

```diff
@@ -584,7 +584,6 @@ void print_result( const fc::variant& result ) { try {
             for( const auto& a : actions ) {
                print_action_tree( a );
             }
-            wlog( "\rwarning: transaction executed locally, but may not be confirmed by the network yet" );
```

Comment:
I think you misunderstood. The idea was not to remove this message, but to find the memory overrun that was causing its output to be garbled.